security/pkg/server/ca/server.go

```diff
@@ -107,7 +107,8 @@ func (s *Server) CreateCertificate(ctx context.Context, request *pb.IstioCertifi
 		sans = []string{impersonatedIdentity}
 	}
 	certSigner := crMetadata[security.CertSigner].GetStringValue()
-	serverCaLog.Debugf("cert signer from workload %s", certSigner)
+	serverCaLog.Infof("generating a certificate for %v, signer: %s, ttl: %d", sans, certSigner,
```

Comment:
This results in a log like

```
2023-05-30T15:32:03.166166Z     info    serverca        generating a certificate for [spiffe://cluster.local/ns/istio-system/sa/istio-ingressgateway-service-account], signer: , ttl: 86400
```

which is kind of confusing. 99% of people have no signer, can we ignore it? TTL should be in time.Duration to properly format. We can also use log.WithLabels to use proper structured logging

Reply:
I kept the `certSigner` because it was there before, and I thought somebody may need it. I will remove it in a later PR.